Sources/HTTP/FoundationClient/Client+Foundation.swift

```diff
@@ -15,12 +15,13 @@ public final class FoundationClient: Client {
     public init(
         scheme: String,
         hostname: String,
-        port: Transport.Port
+        port: Transport.Port,
+        session: URLSession
```

Comment:
You can remove the second `init` by adding a default parameter here:
```
session: URLSession = URLSession(configuration: .default)
```

Reply:
agreed 👍

Reply:
We explicitly avoided a default parameter here

Reply:
@BrettRToomey Why so?